app/controllers/concerns/secured.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Secured
+  extend ActiveSupport::Concern
```

Comment:
I've added some basic comments in this file, which I will flesh out as we begin to use this utility. I haven't thought too much about how it will work with Authz checks. We may have to create a separate concern for that, which will draw upon the Roles/Groups tables we created some months ago